pkg/config/specification.go

```diff
@@ -147,6 +148,7 @@ func init() {
 	viper.SetDefault("web.tls.port", "8444")
 	viper.SetDefault("web.tls.redirect", true)
 	viper.SetDefault("web.credentials.algorithm", "HS256")
+	viper.SetDefault("web.credentials.timeout", time.Hour)
```

Comment:
Pls add this parameter to https://github.com/hellofresh/janus/blob/master/janus.sample.toml as well, just to have it documented.